EECS3311-PD1/src/main/resources/fxml/topMenuView.fxml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.ChoiceBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.ToolBar?>
+<?import javafx.scene.layout.HBox?>
+
+<fx:root type="HBox" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1">
+   <children>
+      <ToolBar prefHeight="40.0" prefWidth="400.0">
+         <items>
+            <Label text="Choose a Country: " />
+            <ChoiceBox fx:id="availableCountries" prefHeight="26.0" prefWidth="63.0" />
```

Comment:
Try to see if you really need the attributes like `prefHeight`, `prefWidth` and others, and remove them if they're redundant